public/api/conf/1.0/schemas/dereferenced/fullReturnSchema.json

```diff
@@ -690,7 +690,7 @@
             "additionalProperties": false
         },
         "submissionType": {
-            "description": "Indicate whether this is the first (original) return or an amended (revised) return",
+            "description": "Indicate if this is a revised return [this should be a boolean true or false]. Limited to the following possible values:",
```

Comment:
I think we need this to be double checked, to me this doesn't make sense. Sounds like you're saying ti can be a boolean or the enum, right?

Reply:
Sorted this with Liam, we're gonna leave it as is whilst it gets checked and we'll raised and bug for it if it's wrong.